spec/feature/guest_user_spec.rb

```diff
@@ -162,6 +162,20 @@
   xit "cannot make themselves an admin" do
   end
 
+  it "can view items with photo next to item" do
+    category = Category.create(name: "Dessert")
+    category.items.create(title: "Bacon Ice Cream", description: "Very delicious", price: 5000, image: "bacon_ice_cream.jpg")
+    click_link_or_button "Menu"
+    within("div.categories") do
+      within("div##{category.name}") do
+        within("div#test") do
+          expect(page).to have_css('img', visible: true)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.